test/support/.percy.yml

```diff
@@ -1,8 +1,16 @@
 version: 1
 snapshot:
   widths: [375, 1280]
-  min-height: 1024
+  min-height: 1024 # px
 static-snapshots:
+  path: _site/
+  port: 9999
   base-url: /blog/
   snapshot-files: '**/*.html'
   ignore-files: '**/*.htm'
+agent:
+  port: 1111
```

Comment:
this is questionable about if this should be here since it's not really supported properly.

Reply:
yeah, I think we should create a follow up ticket to fix it. I've seen a few folks in support try to use it